### PR TITLE
add product card

### PR DESCRIPTION
**Pull request by MartaChobaniuk**
- [DEMO LINK](https://martachobaniuk.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://martachobaniuk.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,60 @@
...
+        <p class="product-card__price-value">$2,199</p>
+      </div>
+
+      <div class="product-card__button-container">
+        <a
+        href="#"
+        class="product-card__button"
+        data-qa="hover"
+        >Buy
+        </a>
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html:56]:
```suggestion
        <a
          href="#"
          class="product-card__button"
          data-qa="hover"
        >
          Buy
        </a>
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+p {
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  background: var(--color-white);
+  border: 1px solid #f3f3f3;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:24]: so move all color if you do it for some of them

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:27]: move font family into html selector

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+  font-style: normal;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:28]: remove redundant default properties

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Hi! Good job👍

1. Check backstop report, there are some inconsistences

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost done) Let's fix tests. Try to use line-height for `product-card__rewiews`
Then switch between two tabs 
image
Test - your solution
Reference - example

Check difference in tests and fix it
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

**Review by vlad-demchuk — APPROVED** [review]
we don't require pixel perfect, however you could try to fix test with this styles for `product-card__rewiews`

```
line-height: 14px;
display: flex;
align-items: center;

**Pull request closed without merging**